content/en/users/check-in/voms/_index.md

```diff
@@ -0,0 +1,291 @@
+geographically and by research institutes. Each CA has its own procedure to
+release certificates.
+
+EGI sites, endpoints and tools accept certificates part of the
+[EUGridPMA](https://www.eugridpma.org/) distribution. If your community VO is
+enabled on that site, your certificate will be accepted by that site since all
+certificates are recognized at site level.
+
+Usually, a certificate can be installed by command line tools, but they can
+also be stored in the web browser to access EGI web tools and services.
+
+### Get a Certificate
+
+The [list of EGI recognised CAs](http://www.eugridpma.org/members/worldmap/)
+provides a clickable map to find your nearby CA.  Several of these offer the
+option to get an 'eScience Personal' certificate online from the Terena
+Certificate Service CA. [Check the countries where this is
+available](https://www.terena.org/activities/tcs/participants.html).
+
+If eScience Personal certificate is not available in your country, then
+request a certificate from a regular [IGTF CA](https://www.igtf.net/pmamap).
+The request is normally generated using either a web-based interface or console
+commands. Details of which type of request a particular CA accepts can be
+found on each CA's website.
+
+For a web-based certificate request, a form must usually be filled in with
+information such as the name of the user, home institute, etc. After submission,
+a pair of private and public keys are generated, together with a request for
+the certificate containing the public key and the user data. The request is
+then sent to the CA, while the private key stays in the browser, hence the
+same browser must be used to retrieve the certificate once it is issued.
+
+Users must usually install the CA root certificate in their browser first.
+This is because the CA has to sign the user certificate using its private key,
+and the user's browser must be able to validate the signature.
+
+For some CAs, the certificate requests are generated using a command line
+interface. The details of the exact command and the requirements of each CA
+will vary and can be found on the CA's website.
+
+Once received the request, the CA will have to confirm your authenticity through
+your certificate. This usually involves a physical meeting or a phone call with
+a Registration Authority (RA). A RA is delegated by the CA to verify the
+legitimacy of a request, and approve it if it is valid. The RA is usually
+someone at your home institute, and will generally need some kind of ID to
+prove your identity.
+
+
+### Install a Certificate
+
+After approval, the certificate is generated and delivered to you.
+This can be done via e-mail, or by giving instructions to you to download it
+from a web page.
+
+#### Browser installation
+
+Install the certificate in your browser. If you don’t know how to upload your
+certificate in your browser have a look at the [examples](https://ca.cern.ch/ca/Help/).
```

Comment:
why this is pointing to CERN?

Reply:
I don't know, it was in the docs provided by @YinChenEGI. I took a look and those docs look quite complete. Do you have a better alternative.

Reply:
I get from the eNMR VO HTC user document:  https://ri-vis.eu/compute/support/egi-platforms/vo-registration-troubleshooting. CERN has quite complete documentation and example for Certificate installation.

Reply:
I don't know a better source for this information.
We could plan to document it later (like by recording an issue in GitHub) but I feel it may OK for now to keep this link.
@andrea-manzi don't be afraid CERN is a trusted source ;)

Reply:
We have things like https://wiki.egi.eu/wiki/USG_Getting_Certificate, but the CERN page may be enough for now

Reply:
Hey, sorry i didn't want to start a long discussion:-) i just was wondering why were pointing to a particular CA instructions, and of course i agree that is quite good documentation:-P